jupyter_rsession_proxy/__init__.py

```diff
@@ -127,6 +150,9 @@ def _get_timeout(default=15):
             'icon_path': get_icon_path()
         }
     }
+    if os.getenv('RSERVER_USE_SOCKET', "") != "":
```

Comment:
Similar to the above, could this please be`JUPYTER_RSESSION_PROXY_USE_SOCKET` instead?

Also, since you check for `unix_socket != ""` above, I think this can be reduced to `if os.getenv("JUPYTER_RSESSION_PROXY_USE_SOCKET")` ?